plugins/krew.yaml

```diff
@@ -1,9 +1,23 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
None of the other plugins includes a license in its manifest. I think we should keep it that way.